packages/manager/apps/hub/src/dashboard/dashboard.html

```diff
@@ -1,6 +1,8 @@
 <div data-ui-view>
     <div class="hub-dashboard-content">
-        <hub-dashboard-welcome></hub-dashboard-welcome>
+        <hub-dashboard-welcome
+            data-is-trusted="$ctrl.me.isTrusted"
```

Comment:
Can't this directly be fetched from `coreConfig.getUser()` ?

Reply:
We are doing that in the route already :)